deployment/live/example-gcp/README.md

```diff
@@ -0,0 +1,24 @@
+## Deployment 
+
+First authenticate via `gcloud` as a principle with sufficient ACLs for
+the project:
+```bash
+gcloud auth application-default login
+```
+
+Then, specify your Google Cloud project ID:
+```bash
+export GOOGLE_PROJECT={VALUE}
```

Comment:
What's the advantage of doing this with environment variables? It moves away from the the deployment files being fully declarative, which means that two people can check out the same repo and then create different deployments because of inconsistencies in the env naming. It's also more work each time you set up a shell to get back to the same state. The obvious workaround is to create a shell script that sets these values, but then you've just put them in a file, which this change moves away from.

Reply:
Huh, I just saw Al's comments which only appear at the overview screen and not on the "files changed" view. Looks like these comments are somewhat redundant!